packages/code-du-travail-frontend/src/outils/DureePreavisRetraite/stepReducer.tsx

```diff
@@ -16,12 +16,19 @@ export const initialState: State = {
   steps: [
     {
       annotation: IntroAnnotation,
-      component: Steps.IntroductionStep,
+      component: () => <Steps.IntroductionStep />,
       label: "Introduction",
       name: "intro",
     },
     {
-      component: Steps.OrigineStep,
+      component: ({ form }) => (
+        <Steps.OrigineStep
+          isVoluntary={
```

Comment:
pourquoi t'as pas choisi de passer mise et faire === "oui" ? pour éviter de faire le not dans le code plus bas

Reply:
En effet, je crois que je l'ai fait dans ce sens car j'ai la traduction pour départ à la retraite mais pas mise à la retraite 😄